lib/manageiq/appliance_console/prompts.rb

```diff
@@ -18,6 +18,7 @@ module Prompts
     SAMPLE_URLS = {
       'nfs' => 'nfs://host.mydomain.com/exported/my_exported_folder/db.backup',
       'smb' => 'smb://host.mydomain.com/my_share/daily_backup/db.backup',
+      's3'  => 's3://mybucket/my_subdirectory/daily_backup/db.backup',
```

Comment:
Not strictly for this PR, but why is this constant defined here? The only place I see it used is in `DatabaseAdmin` outside of the method `sample_url` here, but that's never called outside of specs ... 

@NickLaMuro ?

Reply:
It was here before I was working on `DatabaseAdmin`, so I didn't move it.